malaria24/ona/models.py

```diff
@@ -666,16 +678,19 @@ def get_data(self):
                 #       old format.
                 birth_date = datetime.strptime(self.date_of_birth, '%y%m%d')
 
+            reported_by = ReportedCase.normalize_msisdn(self, self.reported_by)
```

Comment:
You should be able to call it like this:
`reported_by = self.normalize_msisdn(self.reported_by)`

Reply:
I get a `TypeError: unbound method normalize_msisdn() must be called with ReportedCase instance as first argument`
when I don't include `self`. Can you help me get around this?